Android/GoogleDagger/source/Dagger/Transforms/Metadata.xml

```diff
@@ -1,4 +1,8 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <metadata>
-	<attr path="/api/package[@name='com.google.dagger.dagger']" name="managedName">Google.Dagger</attr>
```

Comment:
Do we not have this namespace anymore?

Reply:
That meta is no longer valid. It matches node.